paddle/fluid/inference/tensorrt/plugin/emb_eltwise_layernorm_plugin.cu

```diff
@@ -160,18 +193,32 @@ int EmbEltwiseLayernormPluginDynamic::enqueue(
   const unsigned tpb = 256;
   const dim3 grid(seq_len, batch, 1);
   const dim3 block(tpb, 1, 1);
-  PADDLE_ENFORCE_EQ(
-      out_type == nvinfer1::DataType::kFLOAT, true,
-      platform::errors::InvalidArgument(
-          "The EmbEltwiseLayernorm Plugin only only support fp32 input."));
+  if (sizeof(T) == sizeof(float)) {
+    PADDLE_ENFORCE_EQ(
+        out_type == nvinfer1::DataType::kFLOAT, true,
+        platform::errors::InvalidArgument(
+            "The EmbEltwiseLayernorm Plugin only support fp32 input."));
+  } else {
+    PADDLE_ENFORCE_EQ(
```

Comment:
Is `PADDLE_ENFORCE` works here?

Reply:
Using PADDLE_ENFORCE will result in CI failure

Reply:
done